conf/celeborn-defaults.conf.template

```diff
@@ -24,19 +24,26 @@ rss.rpc.io.numConnectionsPerPeer                 8
 rss.push.io.numConnectionsPerPeer                8
 rss.push.io.threads                              8
 
-rss.worker.flush.queue.capacity                  4096
-rss.push.data.replicate                          true
-rss.metrics.system.enabled                       true
-rss.worker.prometheus.metric.port                9096
-rss.master.prometheus.metric.port                9098
-rss.worker.base.dirs                             /mnt/disk1,/mnt/disk2,/mnt/disk3,/mnt/disk4,/mnt/disk5,/mnt/disk6,/mnt/disk7,/mnt/disk8,/mnt/disk9,/mnt/disk10
+celeborn.push.replicate.enabled                  true
+
+celeborn.metrics.enabled                         true
+celeborn.master.metrics.prometheus.port          9098
+celeborn.worker.metrics.prometheus.port          9096
+
+celeborn.worker.storage.dirs                     /mnt/disk1,/mnt/disk2,/mnt/disk3,/mnt/disk4,/mnt/disk5,/mnt/disk6,/mnt/disk7,/mnt/disk8,/mnt/disk9,/mnt/disk10
+
+celeborn.master.endpoints                        clb-1:9097,clb-2:9098,clb-3:9099
 
-celeborn.master.endpoints                        celeborn-master1:9097,celeborn-master2:9097,celeborn-master3:9097
 celeborn.ha.enabled                              true
-rss.ha.port                                      9872
-rss.ha.service.id                                rss
-rss.ha.nodes.rss                                 node1
-rss.ha.address.rss.node1                         localhost
-rss.ha.storage.dir                               /mnt/disk1/rss_ratis/
-rss.ha.ratis.snapshot.auto.trigger.enabled       true
-rss.ha.ratis.snapshot.auto.trigger.threshold     200000
+celeborn.ha.master.node.1.host                   clb-1
+celeborn.ha.master.node.1.port                   9097
+celeborn.ha.master.node.1.ratis.port             9872
+celeborn.ha.master.node.2.host                   clb-2
+celeborn.ha.master.node.2.port                   9098
+celeborn.ha.master.node.2.ratis.port             9873
+celeborn.ha.master.node.3.host                   clb-3
+celeborn.ha.master.node.3.port                   9099
+celeborn.ha.master.node.3.ratis.port             9874
+celeborn.ha.master.ratis.raft.server.storage.dir                           /mnt/disk1/rss_ratis/
```

Comment:
all ratis configurations respect the ratis raw properties, w/ additional prefix `celeborn.ha.master.ratis.`